data/jibauni/rebel jibauni 0 consultations.txt

```diff
@@ -0,0 +1,359 @@
+				flee
+			label "shot and missed"
+			`	Grabbing a pistol from the inside of the <ship>, you rush outside and fire. Your aim was hasty, however, and the shot goes wildly off-target.`
+				goto speak
+			label "flagship andwerfen"
+			action
+				"reputation: Jibauni (Rebel)" = 10
+			`As you land on this alien planet, a pedastal raises out of the ground about a mile away from the <ship>. A creature with a megaphone is on top of it, and begins yelling something. You faintly hear words telling you to take off and land again, so that they will have time to properly prepare for your arrival. They then use a strange device to launch you into the air.`
+				flee
+			label "met and declined the jibauni and on andwerfen"
+			action
+				"reputation: Jibauni (Rebel)" = 10
+			`A Rebel Jibauni comes up to your ship as you land, telling you to take off and land again so that they can prepare for your arrival. You then are unceremoniously launched by a strange spring.`
+				flee
+			label "met and declined the jibauni"
+			`Upon landing, a Rebel comes up to you telling you to go to <planet> to consult with the leaders. You don't have another option, and so you tell the Jibauni that you will. As the Rebels seem to expect you to leave immediately, you decide to do so.`
+				launch
+			label flee
+			action
+				"reputation: Rebel Jibauni" = -1
+				set "jibauni: disliked by the rebels"
+	on complete
+		event "rebel jibauni ready first contact" 1
+		"reputation: Jibauni (Rebel)" = 10
+		conversation
+			`A Rebel Jibauni tells you to take off and re-land so that they can prepare for your arrival. You are then unceremoniously launched with some strange mechanism.`
+				launch
+
+event "rebel jibauni ready first contact"
+
+mission "Rebel Jibauni: Landed on Andwerfen"
+	landing
+	source Andwerfen
+	to offer
+		or
+			has "First Contact: Rebel Jibauni: declined"
+			has "First Contact: Rebel Jibauni: done"
+		not "attempted to shoot a rebel jibauni"
+		has "event: rebel jibauni ready first contact"
+	on offer
+		conversation
+			`A long wait ensues after you land on <origin>. The Rebel Jibauni seem to be trying to find someone you can speak your language. Feeling bored, you wonder about going to see what's happening.`
+			choice
+				`	(Hey, it's better then staying here the rest of the day.)`
+				`	(You have no idea what this world is about, and don't want to wander unbidden.)`
+					goto stay
+			`	As you wander around the spaceport of <origin>, you notice the unhappy residents make way, even in the thickest crowds, without being asked. At last, you run into a Jibauni who has something around its neck. It gives an exclamation of surprise, and then asks in a language that you can easily understand, "Why have you not come before? I will take you to the government building." It drags you through the city, and then down a side alley. After a long climp up a ramp, you enter a room which has a congregation of dinosaurs.`
```

Comment:
How do we know the residents are unhappy (I don't think dinosaurs can frown)? Congregation of dinosaurs also feels like there's a variety of different dinosaur species. If it's solely Jibuani, maybe something like "a congregation of the dinosaur-like creatures" to more clearly imply it's just the Jibuani.